src/bin/main.rs

```diff
@@ -53,6 +56,18 @@ fn app() -> Rocket {
 }
 
 fn main() {
+    // Set defaults
+    let mut config = Config::default();
```

Comment:
I like the new config system, but it doesn't look like any of the values are actually getting passed to `rocket.rs` right now. We'll need to turn a `config::Config` into a `rocket::Config`, however you'd prefer to do that, then in`fn app()`, replace `let r = rocket::ignite()` with`let r = rocket::custom(<instance of rocket::Config>)`

Reply:
Ok. Sounds good. I'll work on that and add another commit to this PR before we merge it.

Reply:
Thanks :)